packages/ember-routing/lib/services/router.js

```diff
@@ -62,8 +62,24 @@ const RouterService = Service.extend({
        attempted transition
      @public
    */
-  replaceWith() {
+  replaceWith(/* routeNameOrUrl, ...models, options */) {
     return this.router.replaceWith(...arguments);
+  },
+
+  /**
+     Generate a URL based on the supplied route name.
+
+     @method urlFor
+     @param {String} routeName the name of the route
+     @param {...Object} models the model(s) or identifier(s) to be used while
+       transitioning to the route.
+     @param {Object} [options] optional hash with a queryParams property
```

Comment:
The RFC suggests that the argument signature for `urlFor` is `routeName, ...models, queryParams`, however this is proposing `routeName, ...models, { queryParams }` (destructuring `queryParams` off of the last "options" argument).  See [RFC details here](https://github.com/emberjs/rfcs/blob/master/text/0095-router-service.md#new-method-url-generation).  

Why are we diverging?  Did you discover something during implementation that suggests we should have these diverge from the RFC (and no longer have the same API as `transitionTo` and `replaceWith`)?